vendor/zig-yaml/README.md

```
@@ -0,0 +1,163 @@
+This lib is meant to serve as a basic (or maybe not?) YAML parser for Zig. It will strive to be YAML 1.2 compatible
+but one step at a time.
+
+This is very much a work-in-progress, so expect things to break on a regular basis. Oh, I'd love to get the
+community involved in helping out with this btw! Feel free to fork and submit patches, enhancements, and of course
+issues.
+
+
+## Basic installation
+
+The library can be installed using the Zig tools. First, you need to fetch the required release of the library into your project. 
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)